README.md

```diff
@@ -37,9 +34,9 @@ Click the `Ok` button to accept your folder selection.
  Click the `Next` button once the process completes successfully.  
  
 ## Configure CLI  
- ![Configure API Form](img/configure.png)  
- The final step is to enter the Exercism API key which allows you to fetch, keep track of exercises, submit solution etc.  This key is assigned to you when you initially sign up with Exercism.  Use the `I don't have or don't know my API key` link to retrieve your key as needed.  
- Once you have input your API key and have input a `Solution Location`, click the `Configure` button.  This will invoke the Exercism CLI which will be passed the information you just entered.  The output of the CLI will appear in the box at the bottom of the screen (I have blocked out my information).  
+ ![Configure api token](img/configure.png)  
+ The final step is to enter the Exercism api token which allows you to fetch, keep track of exercises, submit solutions etc.  This token is assigned to you when you initially sign up with Exercism.  Use the `I don't have or don't know my token` link to retrieve your token as needed.  
+ Once you have input your token click the `Configure` button.  This will invoke the Exercism CLI which will be passed the token you just provided.  The output of the CLI will appear in the box at the bottom of the screen (I have blocked out my information).  
```

Comment:
>I'd suggest instead of blocking out information, making up a fake token and path

I'll keep this in mind but sounds like it may become moot if the CLI takes care of everything when the user executes the CLI for the first time